src/pybind/mgr/dashboard/frontend/src/app/core/navigation/about/about.component.ts

```diff
@@ -33,4 +49,21 @@ export class AboutComponent implements OnInit, OnDestroy {
   ngOnDestroy(): void {
     this.subs.unsubscribe();
   }
+
+  setVariables() {
+    const project = {} as any;
+    project.hostAddr = window.location.hostname;
```

Comment:
Why not taking the hostname from the mgrmap? That way we will always get the same hostname. With this approach,  if we access the dashboard with `localhost` we'll see here `localhost`, while if we use the IP address (`127.0.0.1`) we'll get here `127.0.0.1`.

Reply:
ack, will amend the PR with that